components/AccountWidget/AccountWidget.jsx

```diff
@@ -23,7 +23,7 @@ const AccountWidget = ({ loading, error, user, style }) => (
     {error && <Error />}
     {!error && !loading && user ? (
       <div>
-        <Link className={styles.tab} to={`/players/${user.account_id}`}>{"Profile"}</Link>
+        <Link className={styles.tab} to={`/players/${user.account_id}/overview`}>{"Profile"}</Link>
```

Comment:
Yea, they are finally merging in a PR I put into material-ui, so I think that will fix that issue but I am not 100% sure.